nsis/src/nsis/format/NsisScriptHeader.java

```diff
@@ -21,8 +21,9 @@
 	static {
 		STRUCTURE = new StructureDataType("script_header", 0);
 		STRUCTURE.add(STRING, NsisConstants.NSIS_MAGIC.length, "magic", null);
-		STRUCTURE.add(DWORD, DWORD.getLength(), "inflated_header_size", null);
 		STRUCTURE.add(DWORD, DWORD.getLength(), "header_size", null);
+		STRUCTURE.add(DWORD, DWORD.getLength(), "length_of_following_data",
+				null);
 		STRUCTURE.add(DWORD, DWORD.getLength(), "compressed_header_size", null);
 		STRUCTURE.add(DWORD, DWORD.getLength(), "flags", null);
```

Comment:
Should this be part of this structure? For a compressed header you don't have this data until it is uncompressed.

Reply:
It's the actual name from the NSIS spec, before it was archiveSize I think!

The weird size that we talked about is actually 'compressed_header_size' and 'flags', I didn't touch those in this PR

Reply:
Understood.